webapp/src/pages/UserPage/UserPage.tsx

```diff
@@ -11,12 +11,13 @@ import {
 } from '../../shared/components';
 import { AVATAR_EP_URL, CHAT_URL } from '../../shared/urls';
 import { useData } from '../../shared/hooks/UseData';
-import React, { useCallback, useEffect, useState } from 'react';
+import { useCallback, useEffect, useState } from 'react';
```

Comment:
why do you remove React from the import list?
When you remove it:
```
'React' refers to a UMD global, but the current file is a module. Consider adding an import instead.ts(2686)`
```

Reply:
Is this an error from the IDE? According to [this](https://stackoverflow.com/questions/64656055/react-refers-to-a-umd-global-but-the-current-file-is-a-module) StackOverflow thread, importing React is no longer required since version 17.

Reply:
It's a symbol we don't use here, so it's totally fine. If I do, for instance, `React.UseEffect(...` then my IDE does complain. I guess my IDE is wrong.